### PR TITLE
views: expose action endpoints

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: invenio_circulation/config.py [hunk 1]
```diff
@@ -97,8 +102,9 @@
                                  ':json_v1_search'),
         },
         list_route='/circulation/loan/',
-        item_route='/circulation/loan/<pid(loan_pid):pid_value>',
+        item_route='/circulation/loan/<{0}:pid_value>'.format(_Loan_PID),
         default_media_type='application/json',
+        links_factory_imp=_CIRCULATION_ACTION_LINKS_FACTORY,
```
Comment (jma) [inline invenio_circulation/config.py:114]: Why do you have to change the `links_factory_imp` for the default REST endpoints?

File: invenio_circulation/pid/fetchers.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 from ..config import _CIRCULATION_LOAN_PID_TYPE
 
 
-def loanid_fetcher(record_uuid, data):
+def loan_pid_fetcher(record_uuid, data):
```
Comment (jma) [inline invenio_circulation/pid/fetchers.py:16]: Should be done in a different commit.

File: tests/test_invenio_circulation.py [hunk 3]
```diff
@@ -19,16 +19,3 @@ def test_version():
     """Test version import."""
     from invenio_circulation import __version__
     assert __version__
-
-
-def test_init():
```
Comment (jma) [inline tests/test_invenio_circulation.py:24]: Why did you remove this?

File: invenio_circulation/config.py (hunk 1 above)
Comment (zzacharo) [reply invenio_circulation/config.py:114]: because when you create a loan it's gonna return back in the links section the next available actions(request, checkout) in order for the ui to decide. In general links will give you back the available actions based on the loans current state.

File: tests/test_invenio_circulation.py (hunk 3 above)
Comment (zzacharo) [reply tests/test_invenio_circulation.py:24]: because it needs other invenio extensions in order to be initialized so it was getting too much for just to check if is in extensions.

File: invenio_circulation/pid/fetchers.py (hunk 2 above)
Comment (zzacharo) [reply invenio_circulation/pid/fetchers.py:16]: you are right, but is it ok for now to leave it as it is because it's not only that if we want to split it in different commits? If it's not I will do it, no worries!

**Review by jma — APPROVED** [review]
